spa/src/oauth/oauthClient.ts

```diff
@@ -55,7 +55,32 @@ export class OAuthClient {
      */
     public async getUserInfo(): Promise<any> {
         
-        return await this.fetch('GET', 'userInfo', null);
+        try {
+
+            // Try the user info call
+            return await this.fetch('GET', 'userInfo', null);
+
+        } catch (e) {
+
+            // Report errors if this is not a 401
+            const remoteError = ErrorHandler.handleFetchError('OAuth Agent', e);
```

Comment:
This didn't work for me. I got a token expired error from idsvr during getUserInfo and a fetch_error was displayed, instead of a refresh.

Reply:
This was a bug with the client side error handling, when a RemoteError was caught that was already a RemoteError. Now fixed and code handles this general condition better.

Reply:
Ah, great.